src/components/shared/CallNodeContextMenu.js

```diff
@@ -143,26 +143,23 @@ class CallNodeContextMenu extends PureComponent<Props, State> {
 
   copyStack(): void {
     const {
-      selectedCallNodeIndex,
+      callNodePath,
       thread: { stringTable, funcTable },
-      callNodeInfo: { callNodeTable },
     } = this.props;
 
-    if (selectedCallNodeIndex === null) {
+    if (callNodePath === null) {
       throw new Error(
         "The context menu assumes there is a selected call node and there wasn't one."
       );
     }
 
-    let stack = '';
-    let callNodeIndex = selectedCallNodeIndex;
-
-    do {
-      const funcIndex = callNodeTable.func[callNodeIndex];
-      const stringIndex = funcTable.name[funcIndex];
-      stack += stringTable.getString(stringIndex) + '\n';
-      callNodeIndex = callNodeTable.prefix[callNodeIndex];
-    } while (callNodeIndex !== -1);
+    const stack = callNodePath
+      .map(funcIndex => {
+        const stringIndex = funcTable.name[funcIndex];
+        return stringTable.getString(stringIndex);
+      })
+      .reverse()
+      .join('\n');
```

Comment:
I think I changed this when I wanted to use only `callNodePath` and get rid of `callNodeIndex` here but I didn't finish it. I can revert this change if you prefer.

Reply:
done !